catroid/src/main/java/org/catrobat/catroid/ui/fragment/FormulaEditorCategoryListFragment.java

```diff
@@ -277,10 +277,9 @@ public void onDismiss(DialogInterface dialogInterface) {
 						}
 					}
 					if (secondSprite != null) {
-						String formula = firstSprite.getName() + " "
-								+ getActivity().getString(itemsIds[pos]) + " " + dialog.getSprite();
+						//String formula = getActivity().getString(itemsIds[pos]) + "(" + dialog.getSprite() +")";
 
-						formulaEditor.addCollideFormulaToActiveFormula(formula);
+						formulaEditor.addCollideFormulaToActiveFormula(secondSprite.getName());
```

Comment:
if this takes a sprite name, then the String parameter name should also be changed.. right now its addCollideFormulaToActiveFormula(String formula)